mysql-plugin/src/test/java/io/cdap/plugin/mysql/MysqlSinkTestRun.java

```diff
@@ -72,96 +120,134 @@ public void testDBSink() throws Exception {
+      }
     }
   }
 
+  /**
+   * Added to prevent 'Ambiguous method call' issue
+   */
+  private void assertObjectEquals(Object expected, Object actual) {
+    Assert.assertEquals(expected, actual);
+  }
+
+  /**
+   * Added to trim arrays of bytes since it's common that actual arrays are larger.
+   */
+  private void assertBytesEquals(byte[] expected, byte[] actual) {
+    Assert.assertTrue(actual.length >= expected.length);
+    Assert.assertArrayEquals(expected, Arrays.copyOf(actual, expected.length));
+  }
+
+  /**
+   * Added to prevent repetitive casts to 'double' and specifying delta.
+   */
+  private void assertNumericEquals(double expected, double actual) {
+    Assert.assertEquals(expected, actual, 0.000001);
+  }
+
+  /**
+   * Added to prevent repetitive casts to 'double' and specifying delta.
+   */
+  private void assertNumericEquals(float expected, float actual) {
+    Assert.assertEquals(expected, actual, 0.000001);
+  }
+
   private void createInputData(String inputDatasetName) throws Exception {
     // add some data to the input table
     DataSetManager<Table> inputManager = getDataset(inputDatasetName);
-    Schema schema = Schema.recordOf(
-      "dbRecord",
-      Schema.Field.of("ID", Schema.of(Schema.Type.INT)),
-      Schema.Field.of("NAME", Schema.of(Schema.Type.STRING)),
-      Schema.Field.of("SCORE", Schema.of(Schema.Type.DOUBLE)),
-      Schema.Field.of("GRADUATED", Schema.of(Schema.Type.BOOLEAN)),
-      Schema.Field.of("TINY", Schema.of(Schema.Type.INT)),
-      Schema.Field.of("SMALL", Schema.of(Schema.Type.INT)),
-      Schema.Field.of("BIG", Schema.of(Schema.Type.LONG)),
-      Schema.Field.of("FLOAT_COL", Schema.of(Schema.Type.FLOAT)),
-      Schema.Field.of("REAL_COL", Schema.of(Schema.Type.DOUBLE)),
-      Schema.Field.of("NUMERIC_COL", Schema.decimalOf(PRECISION, SCALE)),
-      Schema.Field.of("DECIMAL_COL", Schema.decimalOf(PRECISION, SCALE)),
-      Schema.Field.of("BIT_COL", Schema.of(Schema.Type.BOOLEAN)),
-      Schema.Field.of("DATE_COL", Schema.of(Schema.LogicalType.DATE)),
-      Schema.Field.of("TIME_COL", Schema.of(Schema.LogicalType.TIME_MICROS)),
-      Schema.Field.of("TIMESTAMP_COL", Schema.of(Schema.LogicalType.TIMESTAMP_MICROS)),
-      Schema.Field.of("BINARY_COL", Schema.of(Schema.Type.BYTES)),
-      Schema.Field.of("BLOB_COL", Schema.of(Schema.Type.BYTES)),
-      Schema.Field.of("TINYBLOB_COL", Schema.of(Schema.Type.BYTES)),
-      Schema.Field.of("MEDIUMBLOB_COL", Schema.of(Schema.Type.BYTES)),
-      Schema.Field.of("LONGBLOB_COL", Schema.of(Schema.Type.BYTES)),
-      Schema.Field.of("TEXT_COL", Schema.of(Schema.Type.STRING)),
-      Schema.Field.of("TINYTEXT_COL", Schema.of(Schema.Type.STRING)),
-      Schema.Field.of("MEDIUMTEXT_COL", Schema.of(Schema.Type.STRING)),
-      Schema.Field.of("LONGTEXT_COL", Schema.of(Schema.Type.STRING))
-    );
-    List<StructuredRecord> inputRecords = new ArrayList<>();
+    INPUT_RECORDS.clear();
```

Comment:
do not mutate static variables in instance methods

Reply:
Changed.